target/linux/ipq40xx/image/generic.mk

```diff
@@ -709,6 +709,24 @@ define Device/linksys_mr8300
 endef
 TARGET_DEVICES += linksys_mr8300
 
+define Device/linksys_whw03v2
+	$(call Device/FitzImage)
+	DEVICE_VENDOR := Linksys
+	DEVICE_MODEL := WHW03
+	DEVICE_VARIANT := V2
+	SOC := qcom-ipq4019
+	KERNEL_SIZE := 6144k
+	IMAGE_SIZE := 158720k
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	UBINIZE_OPTS := -E 5    # EOD marks to "hide" factory sig at EOF
+	IMAGES += factory.bin
+	IMAGE/factory.bin  := append-kernel | pad-to $$$$(KERNEL_SIZE) | append-ubi | linksys-image type=WHW03v2
+	DEVICE_PACKAGES := ath10k-firmware-qca9888-ct ipq-wifi-linksys_whw03v2 \
+	    -kmod-leds-gpio kmod-leds-pca963x kmod-spi-dev kmod-bluetooth
```

Comment:
I would avoid dropping `kmod-leds-gpio` despite GPIO-s being driven by SSR.

Reply:
I removed it after finding a doc saying that it is better to drop it when using `kmod-leds-pca963x`. 
I will try to find the link of the doc to share it to you. If I don't find it I will put back `kmod-leds-gpio`.

Reply:
Whatever that doc says, it's wrong as without the `gpio-keys` compatible being in the node it's not gonna get used anyway.

I dont see how it can conflict with the PCA963x driver

Reply:
Ok, no problem, I'll put it back. I will do an update probably later today or tomorrow to fix all the comments.
Thanks!

Reply:
done